ezsystems/ezplatform-admin-ui
# EZEE-2062: Unable to embed second item in Richtext field

| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2062 <br /> https://jira.ez.no/browse/EZP-28994 <br /> https://jira.ez.no/browse/EZP-28835
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes/no
| Doc needed?   | yes/no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)


Partially fixes problem. Partially because when you have empty richtext and then:
1. add new embed
2. delete paragraph above

the Add Toolbar will not be displayed. This is because there is no selection region on Ebed field. Style Toolbar is displayed because it is not positioned based on selection.region directly. 

It fixes situation when nothing is selected, but editor has focus. 

#### Checklist:
